mixserv/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,7 @@
+# Set everything to be logged to the console
```

Comment:
I'll try to fix it. 

Having unexpected exclusions (via reflect and resource files) is a drawback of shade plugin :-(